modules/storages/app/common/storages/peripherals/storage_interaction/one_drive/create_folder_command.rb

```diff
@@ -35,31 +35,34 @@ module OneDrive
         class CreateFolderCommand
           using ServiceResultRefinements
 
-          def self.call(storage:, folder_path:)
-            new(storage).call(folder_path:)
+          Auth = ::Storages::Peripherals::StorageInteraction::Authentication
```

Comment:
🟡 Do we need this?

If we declare the namespacing without a shorthand (`Namespace::OtherNamespace` - which I'd dare say is one of the sad antipatterns pushed by RoR), Ruby should be able to infer that `Authentication` would refer to the one under `StorageInteraction` as this is in the path of name resolution.

Reply:
done